src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -825,151 +915,181 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
 				<maml:description>
-					<maml:para>The password for the Windows account used as the service account. If the Service Account was not specified then the account chosen is localsystem and no password is required.  For Active Directory Group Managed Service Accounts no password should be specified.</maml:para>
+					<maml:para>The password for the Service Account.  When using LocalSystem, LocalService,, NetworkService or an Active Directory Group Managed Service Account no password should be specified.</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
+			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
+				<maml:name>AuditRetentionPeriod</maml:name>
+				<maml:description>
+					<maml:para>AuditRetention Period controls how long ServiceControl retains messages ingested from the Audit queue in the database.  The value must be a TimeSpan within the range 1 hour to 365 days.
+Use the New-TimeSpan cmdlet to generate a TimeSpan value.</maml:para>
+				</maml:description>
+				<command:parameterValue required="true" variableLength="false">TimeSpan</command:parameterValue>
+			</command:parameter>
+			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
+				<maml:name>ErrorRetentionPeriod</maml:name>
+				<maml:description>
+					<maml:para>ErrorRetentionPeriod controls how long ServiceControl retains messages ingested from the Error queue in the database.  
+The retention policy does not remove error messages that are in an unresolved state.
+The value must be a TimeSpan within the range 10 days to 45 days.  Use the New-TimeSpan cmdlet to generate a TimeSpan value.</maml:para>
+				</maml:description>
+				<command:parameterValue required="true" variableLength="false">TimeSpan</command:parameterValue>
+			</command:parameter>
+			<command:parameter required="false" variableLength="false" globbing="false" pipelineInput="false" position="named">
+				<maml:name>Force</maml:name>
+				<maml:description>
+					<maml:para>Force the installation even if the directories specified are not empty </maml:para>
+				</maml:description>
+				<command:parameterValue required="false" variableLength="false">SwitchParameter</command:parameterValue>
+			</command:parameter>
 		</command:syntaxItem>
 	</command:syntax>
 	<command:parameters>
 		<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 			<maml:name>Name</maml:name>
 			<maml:description>
-				<maml:para>The name of the ServiceControl Instance</maml:para>
+				<maml:para>The name of the ServiceControl instance.  This will be used as the Windows Service Name.  The name must be unique for all Windows services and not contain any spaces or special characters.</maml:para>
```

Comment:
Seems like a missing word here. Perhaps:

> This will be used as the Windows Service Name.  The name must be unique for all Windows services and can not contain any spaces or special characters.